layouts/partials/footer_section.html

```diff
@@ -1,22 +1,35 @@
-<footer class="site-footer">
-  {{ with .Site.GetPage "privacy.md" }}
-  <p class="powered-by">
-    {{ printf "<a href=\"%s\">%s</a>" .RelPermalink .Title | safeHTML }}
-  </p>
-  {{ end }}
+{{ $year     := now.Year }}
+
+<style>
+.is-cncf-logo {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  padding: 30px;
+}
+</style>
 
-  <p class="powered-by">
-    {{ with .Site.Copyright }}{{ . | markdownify}} &middot; {{ end }}
+<footer class="site-footer">
+  <div>
+    <p style="font-size:larger;">
+      KubeEdge is a <a href="https://cncf.io/">Cloud Native Computing Foundation</a> incubating project.
+    </p>
+    <img class="is-cncf-logo" src="https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/color/cncf-color.png">
+    <p>The Linux Foundation has registered trademarks and
+      uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a
+        href="https://www.linuxfoundation.org/trademark-usage" target="_blank">Trademark Usage</a> page.
+    </p>
+    <p class="has-text-centered">
+      {{ $year}} &copy; KubeEdge Project Authors. All rights reserved.
+    </p>
+  </div>
 
-    
-    {{ if ne .Type "docs" }}
-    <span class="float-right" aria-hidden="true">
-      <a href="#" id="back_to_top">
-        <span class="button_icon">
-          <i class="fas fa-chevron-up fa-2x"></i>
-        </span>
-      </a>
-    </span>
-    {{ end }}
-  </p>
+  <span class="float-right" aria-hidden="true">
+    <a href="#" id="back_to_top">
```

Comment:
The site still uses Hugo. Please do not remove the Hugo code. May be modify the content to add the CNCF requisites.